pkg/agent/params.go

```diff
@@ -124,6 +121,8 @@ func getParameters(config *NodeBootstrappingConfiguration, generatorCode string,
 				addValue(parametersMap, fmt.Sprintf("windowsKeyVaultID%dCertificateStore%d", i, j), c.CertificateStore)
 			}
 		}
+		addValue(parametersMap, "defaultContainerdRuntimeHandler", properties.WindowsProfile.GetWindowsDefaultRuntimeHandler())
+		addValue(parametersMap, "hypervRuntimeHandlers", properties.WindowsProfile.GetWindowsHypervRuntimeHandlers())
```

Comment:
Missing
```


		if !isSetVnetCidrs && properties.HostedMasterProfile != nil && len(agentProfile.VnetCidrs) != 0 {
			// For AKS (properties.HostedMasterProfile != nil), set vnetCidr if a custom vnet is used so the address space can be
			// added into the ExceptionList of Windows nodes. Otherwise, the default value `10.0.0.0/8` will
			// be added into the ExceptionList and it does not work if users use other ip address ranges.
			// All agent pools in the same cluster share a same VnetCidrs so we only need to set the first non-empty VnetCidrs.
			addValue(parametersMap, "vnetCidr", strings.Join(agentProfile.VnetCidrs, ","))
			isSetVnetCidrs = true
		}
```

Reply:
We also need this for Linux case, @yizhang4321